src/v0/destinations/freshsales/transform.js

```diff
@@ -128,6 +128,7 @@ function eventMappingHandler(message, destination) {
   if (!event) {
     throw new InstrumentationError('Event name is required');
   }
```

Comment:
the above validation is already covered in `validateEventName`

Reply:
Removed it.